src/styles/sidebarStyle.tsx

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
why tsx? I think it should just be a .ts file

Reply:
The file extension was changed to .ts